src/main/java/com/github/checkstyle/regression/Main.java

```diff
@@ -34,6 +68,91 @@ private Main() {
      * @throws Exception execute failure
      */
     public static void main(String[] args) throws Exception {
-        // empty for now
+        final Options options = createOptions();
+        final CommandLineParser parser = new DefaultParser();
+        final HelpFormatter formatter = new HelpFormatter();
+        final CommandLine cmd;
+
+        try {
+            cmd = parser.parse(options, args);
+            final String repoPath = cmd.getOptionValue(OPT_REPO);
+            final String branch = cmd.getOptionValue(OPT_BRANCH);
+            final boolean stopAfterGenerateConfig = cmd.hasOption(OPT_STOP_AFTER_GENERATE_CONFIG);
+
+            final File config = generateConfig(repoPath, branch);
```

Comment:
What is `runRegression`? Do you mean create a new method with name `runRegression`?